readme.md

```diff
@@ -164,6 +166,31 @@ csharpgen:
 
 to the autorest configuration to attach a debugger to the plugin process.
 
+### Debugging projects outside `autorest.csharp`
+
+There are two kinds of SDK libraries, one of them is using swaggers as input, the other kind is using TypeSpec files as input.
+
+To debug `autorest.csharp` in a swagger project, you need to add
```

Comment:
Maybe it is a way to debug inside autorest.csharp. and it is both for swagger and typespec.

Reply:
There are two sections here: debugging inside autorest.csharp, and debugging outside autorest.csharp.
Therefore the answer is: yes we have a way to debug inside autorest.csharp, and it works for both swagger and typespec.